core/object/undo_redo.cpp

```diff
@@ -510,6 +514,7 @@ void UndoRedo::_bind_methods() {
 
 	ClassDB::bind_method(D_METHOD("has_undo"), &UndoRedo::has_undo);
 	ClassDB::bind_method(D_METHOD("has_redo"), &UndoRedo::has_redo);
+	ClassDB::bind_method(D_METHOD("is_merging"), &UndoRedo::is_merging);
```

Comment:
You don't need to bind the method, it can be internal.

Reply:
Removed the binding